### DISCUSSION TITLE
[renamerOnUpdate] - Error Help

### DISCUSSION THREAD

**Discussion by noryp5** [stashapp/stash#3440] [Q&A] [answered]
I'll preface this by saying, apologies for being a dumbass. 
I'm running into the following error message:

[renameonupdateerror.txt](https://github.com/stashapp/stash/files/10752308/renameonupdateerror.txt)

I was able to troubleshoot my install using other discussions but I'm not seeing any leads on this. Any help greatly appreciated.

**Comment by DogmaDragon** [comment, marked as answer]
Missing Python module [`requests`](<https://github.com/stashapp/CommunityScripts/tree/main/plugins/renamerOnUpdate#requirement>). 

Run `pip install requests` in cmd/terminal.

**Reply by noryp5** [reply]
Fantastic! Thank you.
